life_expectancy/tests/conftest.py

```diff
@@ -20,4 +20,18 @@ def run_before_and_after_tests() -> None:
 @pytest.fixture(scope="session")
 def pt_life_expectancy_expected() -> pd.DataFrame:
     """Fixture to load the expected output of the cleaning script"""
-    return pd.read_csv(FIXTURES_DIR / "pt_life_expectancy_expected.csv")
+    data_expected = pd.read_csv(
+        FIXTURES_DIR / "pt_life_expectancy_expected.csv"
+    )
+    data_expected["value"] = data_expected[
```

Comment:
why dou you need to do this, don't you want to your expected data to be in the right format from the get go?

Reply:
For some strange reason, it gets to be in a different format than the result of my clean_data

Reply:
This is a common issue with pandas. It will assume the largest sizes for dtypes (int64, float64) which may not be what your actual data is using (int32, float32)